tests/utilities/test_reader.py

```diff
@@ -337,6 +337,29 @@ def test_plot3dmetareader():
         assert all([m.n_points, m.n_cells])
 
 
+def test_multiblockplot3dreader():
+    filename, _ = _download_file('multi-bin.xyz')
+    _download_file('multi-bin.q')
+    reader = pyvista.MultiBlockPlot3DReader(filename)
+    assert reader.path == filename
+
+    mesh = reader.read()
+    for m in mesh:
+        assert all([m.n_points, m.n_cells])
+
+    # Reader doesn't yet support reusability
+    reader = pyvista.MultiBlockPlot3DReader(filename)
+    reader.add_q_files('multi-bin.q')
```

Comment:
A better question for @whophil. IMO this test is more comprehensive than the prior test for this reader since we are calling methods on the vtk object.

Reply:
Perhaps add a call to `reader.read()` after calling `add_q_files()` and assert that point arrays are loaded? (It does work). The test is certainly better than it was before.

Besides that, I don't think we're normally in the business of testing every permutation of the VTK reader class.

Reply:
Now we test the arrays are not loaded without q file and are loaded with q file.